paddle/framework/scope.cc

```diff
@@ -62,5 +64,17 @@ void Scope::DropKids() {
   kids_.clear();
 }
 
+std::once_flag feed_variable_flag;
+
+framework::Scope* GetGlobalScope() {
```

Comment:
return a reference, if this scope doesn't allow to be deleted by callers.

Reply:
Done